api/v1/runtimecomponent_types.go

```diff
@@ -144,6 +148,16 @@ type RuntimeComponentSpec struct {
 	SecurityContext *corev1.SecurityContext `json:"securityContext,omitempty"`
 }
 
```

Comment:
Description is needed here. It'll be shown in OpenShift UI for `serviceAccount` field.
Suggestion: `Defines the service account`